ember/app/components/project-detailed/template.hbs

```diff
@@ -13,10 +13,14 @@
   </div>
 
   <hr class='seperator' />
+  <MaintainerTable @maintainers={{@project.maintainers}} />
   <DependencyTable @versionedDependencies={{@project.versionedDependencies}} />
   <UkButton
     @label='Sync'
     @loading={{this.syncProject.isRunning}}
     @onClick={{perform this.syncProject}}
   />
+  <LinkTo @route="projects.detailed.edit">
+  <UkButton @label="edit"/>
+  </LinkTo>
```

Comment:
A button in a link is not accessible. You can just put the button classes on the link itself:
```suggestion
  <LinkTo @route="projects.detailed.edit" class="uk-button uk-button-default">Edit</LinkTo>
```

Reply:
Fixed here:
https://github.com/adfinis/Outdated/blob/c965aee52e37ef22d4143f91c193b9f67274973a/ember/app/components/project-detailed/template.hbs#L22-L25